torch/_inductor/utils.py

```diff
@@ -720,6 +720,18 @@ def get_fused_kernel_name(
     return "_".join(["fused"] + sources)
 
 
+def _is_split_reduction_node(node: Any) -> bool:
+    """Check if a node is a split reduction node with more than one split."""
+    return (
```

Comment:
I think this would be cleaner as a method on IRNode you override in the needed places.  This would avoid the need for hasattr checks, since the base class will be `return False` then you override the method on the subclasses that have the attribute you are checking for.